meilisearch/models/task.py

```diff
@@ -10,7 +10,9 @@ class Task(CamelBase):
     index_uid: str
     status: str
     type: str
+    canceled_by: Optional[int]
```

Comment:
if `index_uid` is `str` while it can be set to `null`, maybe index_uid should be `Optionnal` as well?